multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.117.0-core.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        logicalFilePath="org/cloudfoundry/multiapps/controller/core/db/changelog/db-changelog-1.117.0-core.xml"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="sap.com" id="alter_operation_table_add_not_null_constraint_current_state">
```

Comment:
Did you check that there is not NULLs at the canary/live landscapes?

In addition, I would refrain in the current tact to introduce the change, if we revert the release.